io.github.piotrek_k._2dTaskBoard.yml

```diff
@@ -0,0 +1,61 @@
+app-id: io.github.piotrek_k._2dTaskBoard
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node20
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --filesystem=home
+build-options:
+  append-path: /usr/lib/sdk/node20/bin
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+modules:
+  - name: electron-sample-app
+    buildsystem: simple
+    build-options:
+      env:
+        XDG_CACHE_HOME: /run/build/electron-sample-app/flatpak-node/cache
+        npm_config_cache: /run/build/electron-sample-app/flatpak-node/npm-cache
+        npm_config_offline: 'true'
+    build-commands:
+      - |
+        cd desktop
+        npm install --offline
+      - |
+        cd web
+        npm install --offline
+        npm run build
+        mkdir -p ../desktop/dist/2dTaskBoard/
+        cp -r dist/* ../desktop/dist/2dTaskBoard/
+      - |
+        . ../flatpak-node/electron-builder-arch-args.sh
+        cd desktop
+        npm run dist -- $ELECTRON_BUILDER_ARCH_ARGS  --linux --dir
+      - |
+        cd desktop
+        cp -a release-builds/linux*unpacked /app/main
+      - install -Dm755 -t /app/bin/ ../run.sh
+      - | 
+        cd desktop/flathub
+        install -Dm644 io.github.piotrek_k._2dTaskBoard.desktop -t /app/share/applications
+        install -Dm644 io.github.piotrek_k._2dTaskBoard.metainfo.xml -t /app/share/metainfo
+        install -Dm644 io.github.piotrek_k._2dTaskBoard.svg -t /app/share/icons/hicolor/scalable/apps
+    subdir: main
+    sources:
+      - type: git
+        url: https://github.com/piotrek-k/2dTaskBoard.git
+        tag: v0.4.1
```

Comment:
Please add the commit for the tag as well.

Reply:
Sure, commit id added